src/shared/components/gene-search-panel/GeneSearchPanel.scss

```diff
@@ -0,0 +1,111 @@
+@import 'src/styles/common';
+
+.geneSearchPanel {
+  height: 100%;
+  background-color: $light-grey;
+  // TODO: Can we find a better name for $global-padding-left? It is not very global, and it is pretty large
+  padding: 65px 0 $global-padding-bottom $global-padding-left; // TODO: the 65px on top looks like it is something repeatable between components?
+  position: relative;
+}
+
+.geneSearchForm {
+  display: grid;
+  grid-template-areas:
+    "label ."
+    "search-field submit-button"
+    "search-scope .";
+  grid-template-columns: 485px min-content;
+  column-gap: 1rem;
+  row-gap: 10px;
+  align-items: center;
+}
+
+.searchLabel {
+  grid-area: label;
+  margin-left: 25px;
+}
+
+.searchField {
+  grid-area: search-field;
+}
+
+.submit {
+  grid-area: submit-button;
+}
+
+.resultsTable {
+  width: 600px;
+  border-collapse: collapse;
+  margin-top: 2rem;
+}
+
+.resultsTable th {
+  text-align: left;
+  border-bottom: 1px solid $blue;
+  padding: 0 0.5rem 0.4rem 1rem;
+  font-weight: $light;
+}
+
+.resultsTable td {
+  padding: 0.4rem 0.5rem 0.4rem 1rem;
```

Comment:
Aren't some of these values used elsewhere for padding and margin? Should these be common values?